web/client/components/mapcontrols/searchservicesconfig/SearchServices.css

```diff
@@ -0,0 +1,403 @@
+/* all this css rules are related to the slider, copy paste from advanced layertree example */
+
+.noUi-target,
```

Comment:
these are generic for every noUi, please make them more specific or move the generic css in proper folders